### DISCUSSION TITLE
Add site upload to snippets

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#1238] [Q&A]
I can never remember how to get an image "into" wave to use elsewhere, would love to see this as a snippet:

```python
(q.app.header_image,) = await q.site.upload(["static/overview.png"])
```

**Comment by mturoci** [comment]
Good suggestion, tracked in: #1239

**Comment by azim-b** [comment]
Also, more importantly, use this "new" path, returned by `q.site.upload`, as the image path to serve the image.
